components/org.wso2.carbon.identity.api.server.configs/org.wso2.carbon.identity.api.server.configs.common/src/main/java/org/wso2/carbon/identity/api/server/configs/common/Constants.java

```diff
@@ -142,6 +150,26 @@ public enum ErrorMessage {
                 "Resource not found.",
                 "Unable to find a resource matching the provided log type %s."),
 
+        /**
+         * DCR Configuration errors.
+         */
+        ERROR_DCR_CONFIG_SERVICE_NOT_FOUND("60509",
```

Comment:
We have provided error codes as **60509**, 65018, 65019, **650290** Are the bold colored error codes added mistakenly?

Reply:
Fixed with [18379bf](https://github.com/wso2/identity-api-server/pull/583/commits/18379bffe485d8f4d642a9dfc7ac2ffc6f1d8fdc)